packages/device-connector/src/deployment-server/connection.ts

```diff
@@ -94,7 +95,8 @@ export const openStream = async (): Promise<WebSocket> => {
   }
 
   socket.onclose = async (event: CloseEvent) => {
-    logger.warn(`Unable to connect to Deployment-Server (Code: ${event.code}) - Reconnecting`)
+    logger.error(`Connection to Deployment-Server failed (Code: ${event.code}) - Trying to reconnect`)
+    await promisify(setTimeout)(3000)
```

Comment:
```suggestion
    await setTimeout(3000)
```

Reply:
resolved by fixed in 65b53609eac25e7ceb18f553e24fee437c18cc93